src/Services/SystemConfigHelper.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopware\Deployment\Services;
+
+use Doctrine\DBAL\Connection;
+
+class SystemConfigHelper
+{
+    public function __construct(private readonly Connection $connection)
+    {
+    }
+
+    public function get(string $key): ?string
+    {
+        $data = $this->connection->fetchOne('SELECT configuration_value FROM system_config WHERE configuration_key = ? AND sales_channel_id IS NULL', [$key]);
+
+        if ($data === false) {
+            return null;
+        }
+
+        return json_decode($data, true, 512, \JSON_THROW_ON_ERROR)['_value'];
```

Comment:
does this always returns a string? :thinking: I guess if you decode it, it could also be an integer or float?